include/devicetree/pwms.h

```diff
@@ -364,6 +437,41 @@ extern "C" {
  */
 #define DT_INST_PWMS_LABEL(inst) DT_INST_PWMS_LABEL_BY_IDX(inst, 0)
 
+/**
+ * @brief Get a pointer to the pwm-controller device object
+ *        from a DT_DRV_COMPAT instance's pwms by index
```

Comment:
```suggestion
 * @brief Get the node identifier for the PWM controller from a
 *        DT_DRV_COMPAT instance's pwms property at an index
```

Looks like this one did not get updated.